index.html

```diff
@@ -401,6 +394,33 @@ <h2>Scrolling APIs</h2>
           </dd>
         </dl>
       </section>
+      <section>
+        <h2>Extensions to the getDisplayMedia algorithm</h2>
+        <p>
+          Extend the
+          <a data-cite="screen-capture#dom-mediadevices-getdisplaymedia"
+            >getDisplayMedia algorithm</a
+          >
+          as follows:
+        </p>
+        <p>
+          Recall that |p| is the promise which the algorithm returns. Immediately before the step
+          which resolves it, add the following steps:
```

Comment:
We haven't had a _"Missing tasks in parallel steps in screen share"_ issue from @dontcallmedom-bot yet, so let's do that first.

Once we've done that, I suggest:
```suggestion
          At the end of the algorithm's [=in parallel=] steps, immediately before
          the step that queues a task to [=resolve=] the promise <var>p</var>,
          append the following steps:
```
This helps highlight the important parts, such as which thread they're to execute on.

Reply:
Please strike my last comment and see https://github.com/w3c/mediacapture-screen-share/issues/312#issuecomment-2655122962 instead.

Reply:
Your PR that fixed that issue has just merged, so I think we're good - resolving.